### DISCUSSION TITLE
Meeting notes - January 9th, 2024

### DISCUSSION THREAD

**Discussion by Chris-ECE** [UNECE/GSBPM_GAMSO_Revision#24] [General]
**Changes to Phase 4**

Echoing the decision to rename phase 4 “Acquire” instead of “Collect” it was agreed also to make the same substitution for the names of the phase 4 subprocesses accordingly.

**BUILD PHASE** 

**Abolish the Build phase & Boundary between Design and Build phases**
•	Juan said you may still need to build tools, and even if some tools for data collection exist, you may still need to customize them for a particular purpose (as opposed to simply configuring them). Corey agreed.
•	Gabriel agreed and cautioned that GSBPM should be applicable also to countries that may be taking a more traditional approach to first designing and then building data collection tools.
•	Noting that the text already refers to “build or reuse”, it was decided not to abolish the build phase.

**Boundary between Build and Collect phases**
It was agreed, based on an earlier discussion to move from 4.2 the bullets from paragraph 70 about Checking detailed information about files, and Arranging secure channels to 3.5.

**Boundary between Build and Disseminate phases** (also covered in Disseminate issues)
•	As the text for 7.1 is about formatting and loading data and doing any remaining final checks, it wasn’t clear what the issue was with this.
•	Carlo suggested it might be a confusion regarding the name of the phase (which includes the word “update”) which may be ambiguous, however, any updating in this subprocess is of data rather than tools.
•	Nadia suggested changing the name “update output systems” to something else.
•	A comment was placed in the document to indicate the subprocess name might be ambiguous.
•	**Action**: Chris asked if anyone has a suggestion for an alternative name to let him know.

**Boundary between Design and Build phases and Outputs from execution of each subprocess** (Ecuador)
•	Producing a generic repository of documents that are output from each subprocess would be out of scope of this revision of GSBPM, however we could potentially include examples in the text if any are apparent.
•	Without further information, it’s hard to tell exactly where the ambiguity lies in differentiating the Design and Build phases.
•	Juan suggested the earlier work on the relation between GSBPM and GSIM might be helpful to address the request for a repository. One option could be to include mention of that work somewhere in the document (maybe near the start).

**Promoting agile development?**
•	This was interpreted not so much as the promotion activity as the mention of agile as a possible development approach in building tools.
•	It was agreed that we should not specify that building tools should be done in an agile way, as this will depend on the preferences and needs of the organization.
•	Some text was added to the descriptions of the Design and Build phases to indicate that some approaches to developing tools and systems involve iteration between these phases.

**Training Machine Learning models** (also covered in Process issues)
•	Based on previous discussion from September, where it was suggested to include training of ML models in the build phase, that question was revisited.
•	Chris pointed out that ML can be used for several different purposes, such as classifying job titles into SOC classification codes, or alternatively for identifying clusters in data or new phenomena (the latter of which are more analytical in nature)
•	Andrew asked whether this implied ML training should be included in multiple places in GSBPM, although another approach might be to rely on the nonlinearity of GSBPM to include it within just one subprocess.
•	Joni said their training could be construed as a design activity, but deploying it is perhaps a build activity.
•	Juan said one option for dealing with training ML models could potentially fit within 3.1 or 3.2. He and Gabriel suggested at making the final decision when the rest of GSBPM has been updated.
•	**Action**: Juan volunteered to draft some text on the training of ML models.
•	Gabriel reminded of the importance of avoiding prescriptive language (e.g. words like “should”) as far as possible.

**Comment by DrJMunozMx** [comment]
Hi all,
Regarding our lasts agreements, I have this initial draft about ML training, please let me know your comments, they will be very welcomed as I think that there is lot of space to improve the text:

Training of Machine Learning Models in GSBPM
ML learning models can be useful for different purposes, related both to the process and how it can be improved and for the information and the new uses we can provide for discovering relations and making analysis.
For the first case, collection of paradata and registering of happenings during the different subprocesses of GSBPM, in special those related to the collect (acquisition) and process phases con be used as valuable inputs to train ML models that can be used to improve the production of information.
The second case can be even more valuable, as the information that has been collected regularly goes to a process of validation and editing that can be improved each time it is performed by capturing the vocabularies and corrections that are needed, after some time, this information can be enough to train ML models to support the automatic validation and editing of information, while not a tool for final results it can help experts with the common tasks letting them more time to expend improving the quality of the data.
Additionally, the statistics itself can be used to train ML models to support new analysis of information that can be the basis for new kinds of information services, like an example, discovering trends and relations of data from different fields.

**Reply by Chris-ECE** [reply]
Thank you Juan! - Am I correctly understanding that you are outlining 3 use examples of:-
- ML (or maybe AI?) for process design/optimisation
- ML for editing and imputation (maybe including automated classification of job titles into occupation codes?)
- ML for statistical analysis (such as identifying clusters in datasets?)

Or where there additional meanings/examples within your proposed text? Thanks.

**Reply by JJLujanS** [reply]
Perhaps a mention can be added about the application of ML in automated coding processes.

**Comment by DrJMunozMx** [comment]
Hi Chris,
Yes, you are right the idea was only to outline examples of uses for ML going apart of only thinking in final outputs as data science techniques can be very useful for the production process too.
Bets regards.

Juan Muñoz López
Director de Planeación y Normatividad Informática
Coordinación General de Informática
Tel. +52-449-910-4332
***@***.******@***.***>

[Imagen que contiene Texto  Descripción generada automáticamente]


De: Chris-ECE ***@***.***>
Enviado el: domingo, 18 de febrero de 2024 06:43 a. m.
Para: UNECE/GSBPM_GAMSO_Revision ***@***.***>
CC: MUÑOZ LOPEZ JUAN ***@***.***>; Comment ***@***.***>
Asunto: Re: [UNECE/GSBPM_GAMSO_Revision] Meeting notes - January 9th, 2024 (Discussion #24)


Thank you Juan! - Am I correctly understanding that you are outlining 3 use examples of:-

  *   ML (or maybe AI?) for process design/optimisation
  *   ML for editing and imputation (maybe including automated classification of job titles into occupation codes?)
  *   ML for statistical analysis (such as identifying clusters in datasets?)

Or where there additional meanings/examples within your proposed text? Thanks.

-
